ui/tests/acceptance/jobs-list-test.js

```diff
@@ -41,7 +41,7 @@ module('Acceptance | jobs list', function (hooks) {
   test('/jobs should list the first page of jobs sorted by modify index', async function (assert) {
     faker.seed(1);
     const jobsCount = JobsList.pageSize + 1;
-    server.createList('job', jobsCount, { createAllocations: false });
+    server.createList('job', jobsCount, { createAllocations: true });
```

Comment:
TODO: figure out why createAllocations was necessary for the job list percy snapshot to show hint bars

Reply:
Answer: it's because of the new `afterCreate` in the job-summary model; data/screenshots are more accurate this way, because that afterCreate correctly shows allocations-that-belong-to-jobs. Before we were just randomly pulling them.